cves/CVE-2015-5964.yml

```diff
@@ -205,10 +209,8 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 2e47f3e401c29bc2ba5ab794d483cb0820855fb9
+    note: Interesting commit, because it seems to address a similar problem to this one, dealing with extra sessions being created on session load.
```

Comment:
Is this just an interesting commit, or could it be considered a VCC or fix?

Reply:
It's a fix in a different section of the code, that I think was inspired from this.